cg/meta/workflow/nf_analysis.py

```diff
@@ -312,22 +313,43 @@ def get_deliverables_template_content() -> list[dict]:
         """Return deliverables file template content."""
         raise NotImplementedError
 
-    def get_deliverables_for_case(self, case_id: str) -> PipelineDeliverables:
-        """Return PipelineDeliverables for a given case."""
-        deliverable_template: list[dict] = self.get_deliverables_template_content()
-        sample_id: str = self.status_db.get_samples_by_case_id(case_id).pop().internal_id
+    def get_deliverables_for_sample(
+        self, sample: Sample, case_id: str, template: list[dict]
+    ) -> list[FileDeliverable]:
+        """Return a list of FileDeliverables for each sample."""
+        sample_id: str = sample.internal_id
+        sample_name: str = sample.name
+        case_path = str(self.get_case_path(case_id=case_id))
```

Comment:
Try to use this with paths, it makes the "/" compatible with the operating system
```suggestion
        case_path = self.get_case_path(case_id=case_id).as_posix()
```

Reply:
Thanks for the suggestion.